integration/kubernetes/README.md

```diff
@@ -0,0 +1,46 @@
+## Pre-requisites:
+
+Alluxio workers use `emptyDir` volumes with `sizeLimit`; which is an alpha feature in Kubernetes 1.8. Please ensure the feature is enabled.
```

Comment:
nit: http://theoatmeal.com/comics/semicolon

Reply:
Thanks for the correction. I'll add docs in a later PR (after HA)